_commands/tools/ping.md

```diff
@@ -0,0 +1,72 @@
+---
+---
+
+ping
+--
+
+`ping` is used to test connections to other machines on a network.
```

Comment:
Worth noting that machines with (or behind) a firewall may not respond to pings. Anyone that doesn't respond to ICMP ECHO_REQUESTs will timeout on ping. Can be configured trivially even in OS X:

<img width="516" alt="screen shot 2016-03-25 at 11 12 44 am" src="https://cloud.githubusercontent.com/assets/5882053/14046881/8e7dd6f0-f27a-11e5-9056-d3d956b72019.png">